asset_dashboard/serializers.py

```diff
@@ -61,6 +61,9 @@ def update(self, instance, validated_data):
 
         phase_data = validated_data.get('phases', [])
 
+        print('phase_data', phase_data)
+        print('instance', instance)
+
```

Comment:
did we want to keep these prints in here for logging in production or was this just for debugging?

Reply:
oop good catch, thanks.